### DISCUSSION TITLE
Who holds the record for being the youngest 🇺🇸 president sworn into office? 🗽

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1275] [Galaxy Trivia] [answered]
**Bonus:** What does the mnemonic "Mary visits every Monday and just stays until noon" help recall?

**Comment by tomerf** [comment]
Theodore  Roosevelt was sworn as president at age 42 and is the youngest person to become president of the United States.

Bonus: The mnemonic is used to remember the order of the planets, as the first letter of each word in the sentence is also the first letter of the matching planet's name in the same order.

**Comment by peterbaccaro** [comment, marked as answer]
The youngest person to become U.S. president was **Theodore Roosevelt**, who, at age 42, succeeded to the office after the assassination of William McKinley.

The mnemonic "Mary visits every Monday and just stays until noon" help us remember the **order of planets orbiting the Sun**.

- Mercury (**M**ary)
- Venus (**V**isits)
- Earth (**E**very)
- Mars (**M**onday)
- Jupiter (**J**ust)
- Saturn (**S**tays)
- Uranus (**U**ntil)
- Neptune (**N**oon)
- Pluto

**Reply by ParanoidUser** [reply]
<img src="https://github.com/ParanoidUser/yolo/assets/5120290/fe638d93-af05-485c-ac15-5b93423bdbcc" width="400" height="400">
